backend/entrypoint.sh

```diff
@@ -31,19 +33,27 @@ done
 
 echo "PostgreSQL started"
 
-
-
 python manage.py migrate
 
 
-if [ "$(python manage.py shell -c 'from accounts.models import User; print(User.objects.filter(username="admin", email="'"$PGADMIN_DEFAULT_EMAIL"'").exists())')" = "False" ]; then
+exists=$(python manage.py shell -c 'import django; django.setup(); from django.contrib.auth import get_user_model; User = get_user_model(); print(User.objects.filter(username="admin").exists())' | tr -d '\n')
+
+if [ "$exists" = "False" ]; then
   echo "Creating superuser..."
-  python manage.py shell -c 'from accounts.models import User; User.objects.create_superuser("admin", "'"$PGADMIN_DEFAULT_EMAIL"'", password="'"$PGADMIN_DEFAULT_PASSWORD"'")'
-  echo "Superuser is created"
+  python manage.py shell -c 'import django; django.setup(); from django.contrib.auth import get_user_model; User = get_user_model(); User.objects.create_superuser("admin", $"PGADMIN_DEFAULT_EMAIL", password=$"PGADMIN_DEFAULT_PASSWORD")'
+  echo "Superuser created"
```

Comment:
This is not a valid bash syntax. The quotes need to surround the variable name and the $. I changed it to this and it works:

```
python manage.py shell -c "import django; django.setup(); from django.contrib.auth import get_user_model; User = get_user_model(); User.objects.create_superuser('admin', '$PGADMIN_DEFAULT_EMAIL', password='$PGADMIN_DEFAULT_PASSWORD')"
```

Reply:
Yes, you are correct I have missed this, but I want to double check why it is working or not in my docker.